indexer/app/lib/indexer_common.rb

```diff
@@ -844,13 +866,10 @@ def configure_doc_rules
 
 
     add_document_prepare_hook { |doc, record|
-      doc['fullrecord'] ||= ''
-      doc['fullrecord'] << IndexerCommon.build_fullrecord(record)
-    }
-
-    add_document_prepare_hook { |doc, record|
-      doc['fullrecord_published'] ||= ''
-      doc['fullrecord_published'] << IndexerCommon.build_fullrecord(record, :exclude_unpublished)
+      if !self.instance_of?(PUIIndexer)
```

Comment:
This is raising a `NameError`, saying that `PUIIndexer` is an uninitialized constant.

Reply:
(pushed a commit to address this)